timetracker/timetracker/settings.py

```diff
@@ -12,7 +12,10 @@
 
 import os
 
-
+os.environ['DJANGO_DEBUG']='true'
+os.environ['DJANGO_DB_NAME']="TimeTracker"
+os.environ['DJANGO_DB_PASSWORD']="Maker42!~"
+os.environ['DJANGO_DB_USER']="postgres"
```

Comment:
Settings should not be committed to the repository. If you want to set environment variables locally, put them in a `.env` file located in the root of the project (ie alongside `Pipfile`). Your `.env` file would look like

```DJANGO_DEBUG=True
DJANGO_DB_NAME=TimeTracker
DJANGO_DB_PASSWORD=Maker42!~
DJANGO_DB_USER=postgres
```

Reply:
yeah I forgot about that to get rid of it when committing whoops my bad.  I tried making my .env file before with pipenv run manage.py runserver and it seemed to load my env vars but did not seem to actually run the local server so i just kept modifying my settings.py instead.